src/seedu/addressbook/common/UtilsTest.java

```diff
@@ -0,0 +1,30 @@
+package seedu.addressbook.common;
+
+import static org.junit.Assert.*;
+
+import java.awt.List;
+import java.util.*;
+
+import org.junit.Test;
+
+public class UtilsTest {
+
+	@Test
+	public void testElementsAreUnique() {
```

Comment:
Note that we have a special naming convention for test method names. Please refer coding standard for more details